tests/plugins/test_plugin.py

```diff
@@ -100,6 +114,10 @@ class AirflowTestPlugin(AirflowPlugin):
     appbuilder_views = [v_appbuilder_package]
     appbuilder_menu_items = [appbuilder_mitem]
     stat_name_handler = staticmethod(stat_name_dummy_handler)
+    global_operator_extra_links = [
+        GoogleLink(),
```

Comment:
how about just put http://Airflow.apache.org as AirflowLink? Not sure if we want to put google Link or facebook Link in our repo?

Reply:
or Kibana log link? Without the internal Lyft knowledge, it isn't very intuitive to user on why we want to put extra link for all the operators.

Reply:
Yeah I think that make more sense to let the user know our intent